server/sonar-web/src/main/js/apps/system/components/info-items/__tests__/__snapshots__/HealthCard-test.tsx.snap

```diff
@@ -8,135 +8,77 @@ exports[`should display the log level alert 1`] = `
 </span>
 `;
 
-exports[`should display the sysinfo detail 1`] = `
-<li
-  className="boxed-group system-info-health-card"
->
-  <div
-    className="boxed-group-header"
-    onClick={[Function]}
-  >
-    <span
-      className="system-info-health-card-title"
-    >
-      <OpenCloseIcon
-        className="little-spacer-right"
-        open={true}
-      />
-      Foobar
-    </span>
-    <HealthItem
-      biggerHealth={true}
-      className="pull-right"
-      health="RED"
-      healthCauses={
-        Array [
-          "foo",
-        ]
-      }
-      name="Foobar"
-    />
-  </div>
-  <div
-    className="boxed-group-inner"
-    onMouseEnter={[Function]}
-    onMouseLeave={[Function]}
-  />
-</li>
-`;
-
 exports[`should render correctly 1`] = `
-<li
-  className="boxed-group system-info-health-card"
->
-  <div
-    className="boxed-group-header"
-    onClick={[Function]}
-  >
-    <span
-      className="system-info-health-card-title"
-    >
-      <OpenCloseIcon
-        className="little-spacer-right"
-        open={false}
+<BoxedGroupAccordion
+  data="Foobar"
+  headerContent={
+    <UNDEFINED>
```

Comment:
🤔

Reply:
Yeah, I tried different ways of writing the headerContent to avoid this, but it seems that the enzyme serializer has still some trouble with fragments...
I think the only way to make it print something better is too create a stateless component for the headerContent. Do you think I should do that ?

Reply:
I guess it can be a function called `renderHeader` instead. We could even pass some parameters if needed.